src/passkey_child/passkey_child_common.c

```diff
@@ -415,6 +427,15 @@ check_arguments(const struct passkey_data *data)
         goto done;
     }
 
+    if (data->action == ACTION_PREFLIGHT
+        && (data->domain == NULL || data->public_key_list == NULL
+        || data->key_handle_list == NULL)) {
+        DEBUG(SSSDBG_OP_FAILURE,
+              "Too few arguments for preflight action.\n");
```

Comment:
AFAIK there is no way to tell what arguments should be provided to `passkey_child --preflight`

`/usr/libexec/sssd/passkey_child --preflight --help` does not say and `/usr/libexec/sssd/passkey_child` only prints `Invalid argument(s).`

Reply:
Same as with other options. In this particular case this option is going to be executed by SSSD and not by the user, so I don't see a reason why we should indicate which options are missing.